### ISSUE TITLE
Multiselect: Accesibility - no label for "toggle all" div and "filter" text box

### ISSUE THREAD

**Issue by markuczy** [primefaces/primeng#14945]
### Describe the bug

MultiSelect component do not have labels set for the following elements.

"Toggle all" div checkbox. Screenshot of the accessibility checker report from documentation (https://primeng.org/multiselect)
![image](https://github.com/primefaces/primeng/assets/129275100/2f624d90-22c5-4b17-83a2-5b73e0f1d252)

"Filter" input textbox. Screenshot of the accessibility checker report from documentation (https://primeng.org/multiselect)
![image](https://github.com/primefaces/primeng/assets/129275100/a0e98949-5fc9-46c3-97e5-5142c022ed12)

### Environment

Ubuntu 22.04, WSL 2, Windows 11 PRO

### Angular version

15.2.10

### PrimeNG version

17.9.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.18.0

### Expected behavior

Expected MultiSelect component to not report any issues with accessibility

**Comment by TaneliTuomola** [comment]
Aria-label for filter text box: https://primeng.org/multiselect#api.multiselect.props.ariaFilterLabel
